.github/workflows/ci.yml

```diff
@@ -109,7 +109,8 @@ jobs:
           it-backend: [local, s3, gcs, minio, azure, azure-hierarchical]
           # IBM not included by default due to lite plan quota being easily exceeded
           #it-backend: [local, s3, gcs, minio, ibm, azure]
-          cassandra-version: [2.2.19, 3.11.11, 4.0.0, 'github:apache/trunk']
+          cassandra-version: [2.2.19, 3.11.11, 4.0.0]
```

Comment:
Controversial? But remove 2.2.19 (or make it just in some simple smoke test) and make 4.1.3 as one of those all-test options? ;)